tests/TestHelpers/TestClock.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Tuf\Tests\TestHelpers;
+
+use Tuf\Helper\Clock;
+
+/**
+ * A test clock class that uses the time from the test fixtures as starting time.
+ */
+class TestClock extends Clock
+{
+    /**
+     * The current time.
+     *
+     * Defaults to the same timestamp used in generate_fixtures.py to create
+     * test fixtures.
+     */
+    private $time = 1577836800;
+
+    public function __construct()
```

Comment:
Now that $this->time has a default value, I don't think we need `__construct()` at all.

Reply:
Removed it. In the future we will provide want to provide different `$time` values to test `timeshoot.json` being expired but `root.json` not being expired but we can change that when write those tests.